lib/src/ui/pages/home/home_page.dart

```diff
@@ -150,8 +150,11 @@ class _HomePageState extends State<HomePage> {
                     final exception = _exceptionMapper.toExceptionType(snapshot.error);
                     return Text(exception.toErrorMessage(context));
                   } else if (snapshot.hasData) {
-                    final article = snapshot.data;
-                    return Text(article.toString());
+                    final articles = snapshot.data;
```

Comment:
można zostawić `return Text(articles.first.toString());` żeby to jakoś wyglądało :P

Reply:
no dupa średnio :D